.github/workflows/pod_failover_test.yml

```diff
@@ -0,0 +1,81 @@
+name: Pod Failover Test
+on:
+    schedule:
+    - cron: "0 13 * * *"
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      AZURE_TENANT_ID: ${{ secrets.TenantId }}
+      AZURE_SUBSCRIPTION_ID: ${{ secrets.SubscriptionId }}
+      AZURE_CLIENT_ID: ${{ secrets.ClientId }}
+      AZURE_CLIENT_SECRET: ${{ secrets.ClientSecret }}
+      AZURE_LOCATION: ${{ secrets.E2ELocation }}
+      REGISTRY: e2etest.azurecr.io/mandalay
+      METRICS_ENDPOINT: ${{ secrets.MetricsEndpoint }}
+    
+    steps:
+    - name: Set UUID
+      id: generate-uuid
+      uses: filipstefansson/uuid-action@v1
+
+    - name: Set Azure resource group
+      run: |
+        echo "AZURE_RESOURCE_GROUP=k8s-e2eTest-${{ steps.generate-uuid.outputs.uuid }}-rg" >> $GITHUB_ENV
+ 
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.16
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install conntrack
+      run: sudo apt install conntrack
+
+    - name: Login to Azure
+      run: |
+        az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET --tenant $AZURE_TENANT_ID
+
+    - name: Login to aks registry
+      run: az acr login --name e2etest
+
+    - name: Build Test
+      env:
+        BUILD_V2: true
+      run: |
+        make container-all push-manifest
+
+    - name: Create an aks cluster 
+      run: |  
+        ./hack/azure-cluster-up.sh --subscription $AZURE_SUBSCRIPTION_ID --location $AZURE_LOCATION --output e2eTestOutput --client-id $AZURE_CLIENT_ID --client-tenant $AZURE_TENANT_ID --client-secret $AZURE_CLIENT_SECRET --resource-group $AZURE_RESOURCE_GROUP
```

Comment:
The default template uses a single-az cluster with two worker nodes. This really isn't enough for a good test. You should create a new temple under `test/e2e/manifest` for the failover tests with a four or five worker node cluster.